www/source/partials/docs/_bp-binary-wrapper-packages.html.md.erb

```diff
@@ -36,17 +36,16 @@ Most binaries compiled in a full Linux environment have a hard dependency on `/l
 2. Invoke `patchelf` on any binaries with this problem during the `do_install()` phase. For example:
 
 ```bash
-patchelf --interpreter "$(pkg_path_for glibc)/lib/ld-linux-x86-64.so.2" \
-         ${pkg_prefix}/bin/somebinary
```

Comment:
Since this is all one command, you'd still need the `\` at the end to split the line, no?  I'm not sure why the `\` was removed.

Reply:
<img width="625" alt="Screen Shot 2019-04-05 at 15 11 22" src="https://user-images.githubusercontent.com/3253989/55659133-256b5900-57b5-11e9-90a9-ad0a48aa700b.png">

I guess this is another one I missed, but yeah, it breaks the site.

Reply:
you also don't want to just put it all on one line because then you get a gross horizontal scrollbar

Reply:
I apologize if my comment wasn't clear.  I agree with you that putting it all on one line is not the answer.

What I meant to say was, when you're displaying an example of shell code, you need the `\` at the end of the line to show that the next line down is a continuation of the same command, and not a new command.  In this particular case, you can put a double backslash `\\` and it will display a single backslash correctly in the rendered code example.  I think that's what we want here, and below.